scipy/stats/stats.py

```diff
@@ -3096,10 +3096,27 @@ def trimboth(a, proportiontocut, axis=0):
     Examples
     --------
     >>> from scipy import stats
-    >>> a = np.arange(20)
-    >>> b = stats.trimboth(a, 0.1)
-    >>> b.shape
-    (16,)
+
+    Create an array of 10 values and trim 10% of those values from each end:
+
+    >>> a = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+    >>> stats.trimboth(a, 0.1)
+    array([1, 3, 2, 4, 5, 6, 7, 8])
+
+    The elements of the input array are trimmed by value, but the output array 
+    is not necessarily sorted:
```

Comment:
I think the example above demonstrates this already. Consider referring to it rather than including a separate example.